extra-scientific/boinc/autobuild/overrides/usr/share/applications/boinc.desktop

```diff
@@ -1,19 +1,22 @@
 [Desktop Entry]
 Type=Application
 Version=1.0
-Exec=/usr/bin/boincmgr
+Exec=/usr/bin/boincmgr -d /var/lib/boinc
 Path=/var/lib/boinc
```

Comment:
`boincmgr` is disconnected because the systemd service is not started. I tried on Xfce, and unless I manually execute `systemctl start boinc.service`, `boincmgr` will not successfully start.